configs/ote_custom_classification/efficientnet_v2_s/template_experimental.yaml

```diff
@@ -0,0 +1,54 @@
+# Description.
+model_template_id: Custom_Image_Classification_EfficientNet-V2-S
+name: EfficientNet-V2-S
+task_type: CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: This model is quite slow, but provides superior single and multi label classification performance.
```

Comment:
I wouldn't call it slow, slower than others (in our templates) in comparison - yes, but generally, it is not so slow, like ResNet50 +-

Reply:
For an SC user these 2 templates is the whole world, so I guess that's true at least at this scale. This template is noticeably slower than MoblieNets.